_tutorials/design_features/Species_Transport/Species_Transport.md

```diff
@@ -0,0 +1,307 @@
+
+## Resources
+
+You can find the resources for this tutorial in the folder [incompressible_flow/Inc_Species_Transport](https://github.com/su2code/Tutorials/tree/master/incompressible_flow/Inc_Species_Transport) and the respective subfolders.
+
+## 1. Restart Validation
+
+The script [restart_validation.sh](https://github.com/su2code/Tutorials/tree/master/incompressible_flow/Inc_Species_Transport/restart_validation.sh) performs 4 simulations using `species3_venturiPrimitive.cfg` to check whether primal and primal-adjoint restarts work. This script is best used with `HISTORY_OUTPUT= RMS_RES` as then the output comparison is the most straight forward. The SU2 binary is deduced from the environmental variable `SU2_RUN`, so make sure that is set correctly. With the explanation given here and the comments in the script it should be straight forward adaptable to other cases.
+1. Primal simulation with n+1 timesteps. This is the ground truth of the expected residual values. Takes Residuals from the `history` file.
+2. Primal simulation with n timesteps. We will restart steps 3. and 4. from this simulation.
+3. Primal simulation with 1 timestep, restarted from simulation in 2nd step. Takes Residuals from the `history` file.
+4. Adjoint simulation with 1 timestep, using the primal restart file from simulation in 2nd step. The primal residuals that are printed in the screen output are taken for comparison.
+
+When using the recommended `HISTORY_OUTPUT= RMS_RES` the output should provide the following: 
+```
+-5.010542647 -4.537626591 -4.207538398 0.4686163065 -6.594021422 0.4978738082 -5.253262361 -5.467591972
+-5.010542647 -4.537626591 -4.207538398 0.4686163065 -6.594021422 0.4978738082 -5.253262361 -5.467591972
+-5.010542647 -4.537626591 -4.207538398 0.4686163065 -6.594021422 0.4978738082 -5.253262361 -5.467591972
+```
+Where in each row the residual of each equation is listed (P, vx, vy, T, k, w, Species_0, Species_0). The primal restart (2nd row) and the adjoint primal restart (3rd row) provide identical results compared to the 'full' primal simulation (1st row). Small deviations in the last digits are not an issue, especially when higher iteration counts are used (here only 10). But if the adjoint restart provides a clearly different result then this should be debugged before attempting a gradient validation or even optimization.
+
+The config option `OUTPUT_PRECISION= 16` can be set to compare more digits if necessary.
+
+Execute the scipt by:
+```
+$ bash restart_validation.sh
+```
+
+## 2. FFD-Box Setup
+
+The setup is fairly simple when following some simple rules. The additional block of code necessary to write the FFD box is given below. Essentially, there are only 3 options (`FFD_DEFINITION`, `FFD_DEGREE` and `DV_MARKER`) where user input is necessary. `DV_KIND= FFD_SETTING` and `DV_PARAM= ( 1.0 )` are fixed and not to be changed.
```

Comment:
Great stuff, what do you think of giving this info some more visibility? For example linking to it somewhere at the top of "Summary of tutorials"?

Reply:
Or mention this section (and link it) in the 3 Design Tutorials that use FFD-boxes. The OneraM6, inv_wedge and unsteady shape opt

Reply:
`The order is important, and you can use the example above to match the convention.` ... Thanks for nothing OneraM6 Tutorial :D

Reply:
ca27520  Now anyone who comes across a tutorial that uses an FFD-box will be pointed to this.